corgi-editor/corgi-editor.el

```diff
@@ -26,15 +28,20 @@
 (use-package counsel
   :after (ivy)
   :config
+  (counsel-mode)
   ;; This ensures that SPC f r (counsel-recentf, show recently opened files)
   ;; actually works
   (recentf-mode 1))
 
-;; Make counsel-M-x show most recently used commands first
-(use-package smex)
+(use-package ivy-prescient
+  :after (ivy)
+  :config
+  (ivy-prescient-mode 1)
+  (prescient-persist-mode 1))
 
 (use-package swiper
-  :after (ivy))
+  :after (ivy)
+  :bind (("C-s" . swiper)))
```

Comment:
Let's bind this with corgi-keys/corgi-signals through a signal.

Reply:
We already have 

>:buffer/incremental-search swiper

in corgi-signals.el and bound to SPC s s in corgi-keys.el. Do you mean to also bind it globally to C-s in corgi-keys.el?

Reply:
On second thought I'm not sure we should have an opinion about `C-s` at all. We already have `/`, `?`, `SPC s s`. Emacs bindings like `C-s` are from Corgi's point of view legacy bindings. They happen to work and that's nice to help people transition, but it's basically by coincidence, and we retain the right to use that key combination for something else in the future (not too likely in this case since it's a key chord, but in principle we do).

Reply:
Seems fair enough. It is pretty trivial for users to add it in their user-keys.el if they want it.